.github/workflows/aws.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       ORY_SDK_URL: ${{ secrets.ORY_SDK_URL }}
       ORY_ACCESS_TOKEN: ${{ secrets.ORY_ACCESS_TOKEN }}
       ORYCLOUD: "enabled"
+      ENABLE_BANNER_DONATION: ${{ env.NEXT_PUBLIC_ENABLE_BANNER_DONATION }}
```

Comment:
question: should it be `vars` instead of `env`?

Reply:
it has to be vars, because that's how we're going to use it in github actions. I got confused